rfcs/README.md

```diff
@@ -0,0 +1,108 @@
+# Beneficary FHIR Server RFCs
+[header-and-summary]: #header-and-summary
+
+Each "Request for Comment (RFC)" is a proposal for a substantial change to our systems and/or processes.
+The RFC process is intended to provide some formalization around such proposals,
+  allowing the entire team a chance to review them and to have a voice in such decisions before they're made.
+It's also intended to democratize the process: anyone can create an RFC.
+
+Pragmatically: such decisions need review and input, and GitHub pull requests are a great system for enabling that.
+
+## Table of Contents
+[table-of-contents]: #table-of-contents
+
+* [Beneficiary FHIR Server RFCs](#header-and-summary)
+* [Table of Contents](#table-of-contents)
+* [When Is An RFC Needed](#when-is-an-rfc-needed)
+* [Before Creating an RFC](#before-creating-an-rfc)
+* [The RFC Process](#the-rfc-process)
+    * [Approval](#approval)
+    * [Postponement](#postponement)
+    * [Implementation](#implementation)
+* [Inspiration](#inspiration)
+
+## When Is an RFC Needed?
+[when-is-an-rfc-needed]: #when-is-an-rfc-needed
+
+What counts as "substantial"? Things like the following:
+
+* A major new feature.
+* Deprecation or removal of existing features.
+* Most changes that would require a Security Impact Aseessment (SIA).
+* Major changes to our processes, particularly any that might break team members' existing workflows.
+
+When (reasonably) in doubt, create an RFC first, as the lack of one might hold up merging your PRs.
+
+## Before Creating an RFC
+[before-creating-an-rfc]: #before-creating-an-rfc
+
+Things to do _before_ drafting your RFC:
+
+1. Have a conversation with your company or program Information System Security Officer (ISSO) about your proposal.
+    1. Which parts of the proposal might be _For Official Use Only (FOUO)_? Those parts will need to be excluded, or instead put into a separate, encrypted addenda.
+    2. Which parts of the proposal might be _System Configuration Management Information (SCMI)_?
+       Those parts will need to be excluded, or instead put into a separate, encrypted addenda.
+2. Have a conversation with your company and program tech leads.
+    1. Would they be likely to support your proposal?
+    2. If not, what objections would you need to overcome to get them to vote in favor of it?
+3. Have a conversation with other relevant program and/or community members.
+    1. Would they be likely to support your proposal?
+    2. If not, what objections would you need to overcome to get them to vote in favor of it?
+
+There'a no harm in throwing up an RFC as a "trial balloon" but a much faster way to get feedback on an idea first... is to _talk about it_ with others.
+Accordingly, you ought to have these conversations (e.g. in Slack) first, **before** publishing your RFC.
+
+## The RFC Process
```

Comment:
What is the RFC process?

Reply:
This was pretty confusing, I think, but the process was detailed in the early step in the "Approval" section.

I restructured these sections to be more clear. Do they make sense to you now?